webix/index.d.ts

```diff
@@ -212,7 +212,7 @@ function message(content:any):void;
 function modalbox(text:string, callback:WebixCallback):HTMLElement;
 function once(code:WebixCallback):void;
 function proto(target:any, ...mixins:any[]):any;
-function protoUI(target:any, view:any, mixins:any):any;
+function protoUI(target: any, ...mixins: any[]): any;
```

Comment:
It's useful for `view` to be present for the sake of documentation and clarity when using the `.d.ts` file.